spec/rspec/rails/example/helper_example_group_spec.rb

```diff
@@ -42,7 +42,11 @@ class InternalClass
         group = RSpec::Core::ExampleGroup.describe do
           include HelperExampleGroup
           def _view
-            ActionView::Base.new
+            if ActionView::Base.respond_to?(:empty)
```

Comment:
@benoittgt we typically define methods conditionally for this sort of thing, fancy refactoring it in the dev branch?

Reply:
I am sorry @JonRowe I didn't see your message. What do you mean by "define methods conditionally" in that case?

Thanks

Reply:
```Ruby
if ActionView::Base.respond_to?(:empty)
  def _view
    ActionView::Base.empty
  end
else
  def _view
    ActionView::Base.new
  end
end
```

Reply:
Ohhh ok. Thanks for your patience. 

I made the change in https://github.com/rspec/rspec-rails/pull/2071/commits/bf65b86556c2bdd152962ac5ad17c4dcbb74b534